src/renderer/components/icon/icon.tsx

```diff
@@ -12,7 +12,57 @@ import type { LocationDescriptor } from "history";
 import { cssNames } from "../../utils";
 import { withTooltip } from "../tooltip";
 import isNumber from "lodash/isNumber";
-import { decode } from "../../../common/utils/base64";
+import Configuration from "./configuration.svg";
+import Crane from "./crane.svg";
```

Comment:
Probably an idea for the other refactoring but I would create separate React component for each of these icons containing an svgs inside. Readability and typings would also benefit from that. For example to use:

`<CraneIcon/>` instead of `<Icon svg="crane"/>`

And we probably should remove `svg` attribute form the `Icon` completely in the future.

Reply:
Yes I agree